pkg/config/config.go

```diff
@@ -156,6 +156,9 @@ func init() {
 	Datadog.SetDefault("kubelet_client_crt", "")
 	Datadog.SetDefault("kubelet_client_key", "")
 
+	Datadog.SetDefault("kubernetes_collect_service_tags", true)
```

Comment:
We should probably bindenv them, and add them (with extensive documentation) in the config template